src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -278,9 +284,65 @@ pServiceApiConfig = id
     <*< serviceApiConfigInterface .:: textOption
         % prefixLong service "interface"
         <> suffixHelp service "interface that the service Rest API binds to (see HostPreference documentation for details)"
+    -- serviceApiBackups isn't supported on the command line
   where
     service = Just "service"
 
+-- -------------------------------------------------------------------------- --
+-- Backup configuration 
+
+data BackupApiConfig = BackupApiConfig 
+    { _backupApiDirectory :: !FilePath
```

Comment:
This is an absolute path or relative to the current working dir, but it should in fact be inside the same partition as the database directory or the rocksdb snapshot will revert to its slow mode, copying the files instead of hardlinking. Will comment.